docs/sources/CHANGELOG.md

```diff
@@ -57,6 +57,7 @@ The CHANGELOG for the current development version is available at
 - Raises a more informative error if a `SparseDataFrame` is passed to `apriori` and the dataframe has integer column names that don't start with `0` due to current limitations of the `SparseDataFrame` implementation in pandas. ([#503](https://github.com/rasbt/mlxtend/pull/503))
 - SequentialFeatureSelector now supports DataFrame as input for all operating modes (forward/backward/floating). [#506](https://github.com/rasbt/mlxtend/pull/506)
 - `mlxtend.evaluate.feature_importance_permutation` now correctly accepts scoring functions with proper function signature as `metric` argument. [#528](https://github.com/rasbt/mlxtend/pull/528)
+- Fixed documentation of `iris_data()` under `iris.py` by adding a note about differences in the iris data in R and UCI machine learning repo.
```

Comment:
Sorry, this is maybe a bit of an awkward timing, but I just made a new release version this morning. Could you add this line to the 0.17.0dev0 section at the top of the document instead?

Reply:
Hi @rasbt, I have made the required changes.